examples/pruner/job.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: batch/v2alpha1
+kind: ScheduledJob
+metadata:
+  name: image-prune
+spec:
+  schedule: 0/15 * * * ?
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: prune
+            image: openshift/origin:latest
+            args: [ "admin", "prune", "images", "--confirm"]
+          restartPolicy: OnFailure
```

Comment:
@miminar i would rather restart on failure... if you specify Never and something goes horribly wrong, what you will see is the job creating pods like crazy... cc'ing @soltysh :)